README.adoc

```diff
@@ -951,7 +951,9 @@ end
 
 === Context Descriptions
 
-`context` block descriptions should always start with 'when', 'with', or 'without' and be in the form of a sentence with proper grammar when composed with `it` block descriptions.
+When an `it` block description is composed with a `context` block description, it should result in a sentence with
+proper grammar. This typically means the `context` should start with a term such as 'when', 'with', or 'without', and
+should have a clear distinction from other contexts.
```

Comment:
This statement seems to fit better the [Context Cases guideline](https://github.com/rubocop-hq/rspec-style-guide#context-cases).

Reply:
I've chosen to remove this part, since it seems obvious. WDYT?